src/modules/file-transfer-service/FileTransferService.ts

```diff
@@ -120,15 +101,22 @@ export class FileTransferService {
         }
     }
 
-    private getErrorFrom(err: any, fileId: string): Error {
-
-        if (err.isAxiosError) {
-            switch (err.response.status) {
-                case NOT_FOUND:
-                    return new Error(`File ${fileId} not found.`);
+    private handleResponseError(operation: 'upload' | 'metadata retrieval' | 'download' | 'deletion', subject: string):
+        (err: AxiosError) => never {
+        // tslint:disable: max-line-length
+        return (err: AxiosError) => {
+            if (err.isAxiosError) {
+                switch (err.response!.status) {
+                    case FORBIDDEN:
```

Comment:
To me, 'FORBIDDEN' means unauthorised, not 'in progress'....however i don't really know of a more suitable error code for this

Reply:
Well this is not in my hands, this is how API behaves: https://github.com/companieshouse/file-transfer-api/blob/master/src/routes/file-download.ts#L16

I basically went through an API code and reflected all scenarios in the service code.

Reply:
ok, fair enough 👍